cypress.config.js

```diff
@@ -1,8 +1,14 @@
 const { defineConfig } = require('cypress');
 
+
 module.exports = defineConfig({
   e2e: {
     setupNodeEvents(on, config) {
-    }
-  }
+     
+    },
+    env: {
+      email: 'mate8568@ukr.net', 
+      password: 'Q1w2e3r4',      
```

Comment:
Hardcoding sensitive information like email and password directly in the code is a security risk. This information could be leaked if the codebase is compromised. Instead, consider using environment variables to store sensitive data. You can use the `Cypress.env` method to access these variables.